.packit.yaml

```diff
@@ -567,84 +847,166 @@ jobs:
       - tier1
       - rocky-8
 
+    require:
+        label:
+          present:
+            - tests-run-tier0
+            - tests-run-tier1
+            - tests-run-sanity
+            - tests-run-all
+          absent:
+            - tests-skip
+
   ## Tests on merge to main stage. Tests are run automatically
   - &tests-main-tier1-centos
     <<: *tests-tier0-non-destructive-centos
-    # Run test automatically with merge commit to main branch
-    manual_trigger: false
     identifier: "tier1-centos"
     tmt_plan: "tier1"
     trigger: commit
     branch: main
 
+    require:
+        label:
+          present:
+            - tests-run-tier0
+            - tests-run-tier1
+            - tests-run-sanity
+            - tests-run-all
+          absent:
+            - tests-skip
+
```

Comment:
This should run automatically on a merge commit to main.
We don't need the label trigger here.
The same goes for the below.
```suggestion
```